docs: add site-admin capabilities (#8285)
sourcegraph/sourcegraph: doc/admin/capabilities.md

> # Site administrator capabilities

Reviewer: ```suggestion
# Site administrator privileges
```

> Site administrators have full administrative access to the Sourcegraph instance. In many cases, they also control the deployment environment. Therefore, it is reasonable to have extra capabilities for site administrators.

Reviewer: ```suggestion
Site administrators have full administrative access to the Sourcegraph instance. In many cases, they also control the deployment environment. Special privileges are granted to site-admin users.
```

## Access to all repositories

> Site administrators are able to access all repositories added to the Sourcegraph instance, and manage settings of individual repository, including repository indexing status, statistics, mirroring information and code intelligence assets.

Reviewer: ```suggestion
Site administrators are able to access all repositories on the Sourcegraph instance and manage the settings of individual repositories.
```

## Access to all GraphQL APIs

> Site administrators are able to access all GraphQL APIs of the Sourcegraph instance. There are many GraphQL APIs require the client to have the token from a site administrator.

Reviewer: ```suggestion
Site administrators are able to access all GraphQL APIs of the Sourcegraph instance, including special GraphQL APIs that require a site-admin access token.
```

## Impersonate regular users

Site administrators are able to impersonate regular users in two ways:

> 1. View and update settings of individual user.

Reviewer: ```suggestion
1. View and update settings of individual user.
```
```suggestion
1. View and update the settings of any user.
```

> 2. Initiate GraphQL API calls on behave of a regular user.

Reviewer: ```suggestion
2. Initiate GraphQL API calls on behalf of any other user.
```

## Receive site alerts

> Site administrators are able to see site alerts in the form of top banner about configuration problems of the Sourcegraph instance.

Reviewer: ```suggestion
Site administrators see update notifications and other site-level alerts (visible as a banner across the top of the screen) that may be invisible to non-admin users.
```
